core/src/i18n/en.yaml

```diff
@@ -558,6 +559,7 @@ deployments:
       image-pull-secret: Image Pull Secret
       image-pull-secret-tooltip: The secret for authorization to the private Docker repository.
     title: Create Deployment
+  description: ' is a Kubernetes object that represents a replicated application running on your cluster.'
```

Comment:
Do you mean `application` in a general sense, or `Application`, as in our custom resource representing an external solution?

Reply:
According to my understanding it is general.